spec/services/alerts/importer_spec.rb

```diff
@@ -4,17 +4,15 @@
 
 RSpec.describe Alerts::Importer do
   subject(:importer) do
-    described_class.new(
-      profile: profile,
-      alerts: alerts
-    )
+    Moves::Importer.new(moves)
```

Comment:
This looks like it changes the importer being tested from `Alerts::Importer` to `Moves::Importer`. Is that intentional, and if so should the test be moved/renamed to reflect it?

Reply:
Yes its intentional. No the test shouldn't be renamed, as it's goal is still to test the Alerts part of the importer. The original test was poor because it tested the Alerts importer in isolation and so failed to find several defects. Configuring the Moves::Importer with a focus on the Alerts::Importer rather than creating a 'naked' Alerts::Importer results in a much more effective test